cmd/podman/runlabel.go

```diff
@@ -152,7 +152,7 @@ func runlabelCmd(c *cliconfig.RunlabelValues) error {
 		return err
 	}
 	if !c.Quiet {
-		fmt.Printf("command: %s\n", strings.Join(cmd, " "))
+		fmt.Printf("command: %s\n", strings.Join(append(os.Args[0], cmd[1:]), " "))
```

Comment:
Tests are complaining that the first argument to `append` must be a slice, so I think you'll need to make it `append([]string{os.Args[0]}, cmd[1:]...)`

Reply:
Guilty as charged, dry-coded :)